Makefile.macros

```diff
@@ -102,7 +102,7 @@ endef
 #   - foo_LIBS:
 #     extra libraries built in this project linked into the binary
 #     (just specify the name without extension)
-#   - foo_EXTRA_LDFLAGS:
+#   - foo_LDFLAGS:
```

Comment:
apologies that I made this inconsistent, but the original reasoning was something like this:
- there may be a global LDFLAGS, and it might at times be necassary to inject something into it from outside the build system for special builds; these might then need to be honored for all targets
- at the same time, binaries and tests might link to external libraries -- they might not wish to "override" the global LDFLAGS, but amend to them; hence the intent as "extra LDFLAGS" ("add to" global LDFLAGS, not "override" global LDFLAGS)
However I admit that this got slightly confused when putting together the macros. Given that there is still TEST_EXTRA_LDFLAGS, maybe they should then at least be made consistent in their meaning? I don't mind dropping the "EXTRA" portion at all, maybe I should not have put that in.

Reply:
Renamed to `*EXTRA_LDFLAGS`. Could you have a second look, please?